src/views/layout.njk

```diff
@@ -46,6 +46,20 @@
       html: 'This is a new service – your <a class="govuk-link" href="https://www.research.net/r/alfp-feedback" target="_blank">feedback</a> will help us to improve it.'
     })
   }}
+
+  {% if userEmail %}
+    <div class="govuk-section-break--visible">
+      <div class="sign-out-user-banner">
+        <span id="signed_in_user_email" class="sign-out-user-banner__email">
+          {{ userEmail }}
+        </span>
+        <span>
+          <a id="sign_out" href="/signout">Sign out</a>
+        </span>
+      </div>
+    </div>
```

Comment:
consider making this a template to keep the layout file as light as possible.

Reply:
Yeah will do 👍 was just pushing up the main functionality first